.github/workflows/ping-interop-latest.yml

```
@@ -14,22 +14,22 @@ on:
         required: false
         type: string
       custom_interop_target:
-        description: in the case of cross-implementation testing, the implementation target (go | rust | ...)
+        description: in the case of cross-implementation testing, the implementation target (go | rust | nim)
         required: false
         type: string
   push:
   pull_request:
 
-name: libp2p ping - go + rust test (latest) with testground.
+name: libp2p ping - go + rust + nim test (latest) with testground.
```

Nit pick, while we are at it, how about being consistent with the naming on `ping-interop.yml`?
```suggestion
name: libp2p ping - go, rust and nim test (latest) with testground.
```